jupyter-book/preamble.md

```diff
@@ -12,53 +12,57 @@ Interested in contributing? Please have a look at https://github.com/theislab/si
 
-This monumental task is approached in different ways of which the most promising one is to profile cells at the individual level. So far, each cells' transcriptome was primarily examined in a process known as single-cell RNA sequencing. With recent advances in single-cell genomics, it is now possible to enrich the transcriptome information with spatial, chromatin accessibility or protein information. These advances generate not only insight into complex regulatory mechanisms, but also go along with additional complexity for data analysts.
+To address this challenge, researchers employ a variety of strategies, with one of the most promising being profiling cells at the individual level. Traditionally, each cells' transcriptome was primarily examined in a process known as single-cell RNA sequencing. However, recent advancements in single-cell genomics now enable the integration of transcriptome data with spatial, chromatin accessibility, or protein-level information. These developments not only enhance our understanding of complex regulatory mechanisms but also introduce additional challenges in data analysis.
 
-Nowadays, data analysts are facing a vast analysis tool landscape with a collection of more than 1000 computational single-cell analysis methods. It is becoming increasingly challenging to navigate this large range of different tools to generate sound results which are at the forefront of science.
+Currently, analysts are faced with an overwhelming array of computational tools - over 1,000 methods dedicated to single-cell analysis alone. Navigating this extensive landscape to produce reliable, cutting-edge results poses a significant challenge.
 
 ## What this book covers
 
-The goal of this book is to teach newcomers and advanced professionals alike, the **best practices** of single-cell sequencing analysis.
-This book will teach you the most common analysis steps ranging from preprocessing to visualization to statistical evaluation and beyond. A read through the entire book will enable you to analyze unimodal and multimodal single-cell sequencing data on your own. The guidelines and recommendations in this book are not only tailored to teach you how to do single-cell analysis in general, but how to do them right. We base our suggestions on external benchmarks and reviews whenever possible. Finally, we consider this book to be a living resource for single-cell data analysts which can easily be updated when the recommendations change.
+This book aims to guide both beginners and experienced professionals in **best practices** of single-cell sequencing analysis. It provides a comprehensive overview of essential analytical steps, from preprocessing to visualization and statistical evaluation and beyond. By following this book, readers will acquire the skills to independently analyze unimodal and multimodal single-cell sequencing data.
```

Comment:
```suggestion
This book aims to guide both beginners and experienced professionals in **best practices** of single-cell sequencing analysis.
It provides a comprehensive overview of essential analytical steps, from preprocessing to visualization and statistical evaluation and beyond.
By following this book, readers will acquire the skills to independently analyze unimodal and multimodal single-cell sequencing data.
```

Style wise, should we say "the reader" or YOU? I think the latter is probably more common and better?